tests/unittests/test_client.py

```diff
@@ -1,10 +1,12 @@
+import pexpect
```

Comment:
Please all pexpect related test under `cli_test`, since it doesn't belong to unittest.

Thanks one more fix then I will merge :)